Source/Shaders/BillboardCollectionVS.glsl

```diff
@@ -246,8 +246,8 @@ void main()
 #endif
 
 #ifdef DISTANCE_DISPLAY_CONDITION
-    float nearSq = distanceDisplayCondition.x * distanceDisplayCondition.x;
-    float farSq = distanceDisplayCondition.y * distanceDisplayCondition.y;
+    float nearSq = distanceDisplayConditionAndDisableDepth.x * distanceDisplayConditionAndDisableDepth.x;
```

Comment:
Unrelated to this PR, but why don't we store the squared distance in the vertex attribute to avoid this computation?